src/xrc/xh_sizer.cpp

```diff
@@ -253,6 +253,13 @@ wxObject* wxSizerXmlHandler::Handle_sizer()
     m_isInside = true;
     m_isGBS = (m_class == wxT("wxGridBagSizer"));
 
+    if (m_isGBS)
+    {
+        wxSize cellsize = GetSize(wxT("empty_cellsize"));
+        if (cellsize != wxDefaultSize)
+            wxDynamicCast(sizer, wxGridBagSizer)->SetEmptyCellSize(cellsize);
```

Comment:
I'd use `static_cast<>` here because if the object dynamic type is not `wxGridBagSizer` we're going to crash anyhow.